### PR TITLE
Synchronized _ru.properties with origin properties

### PR DESCRIPTION
**Pull request by kolosov-sergey**
I've found missed properties in Russian translation and added them into appropriate files

### CODE REVIEW

File: src/web/core/src/main/resources/GeoServerApplication_ru.properties [hunk 1]
```diff
@@ -12,18 +12,22 @@ AboutGeoServerPage.moreInformation = \u0411\u043e\u043b\u044c\u0448\u0435 \u0438
...
 AboutGeoServerPage.title         = \u041e GeoServer
+AboutGeoServerPage.usefulLinks   = Useful Links:
 AboutGeoServerPage.userMap       = \u041a\u0430\u0440\u0442\u0430 \u043f\u043e\u043b\u044c\u0437\u043e\u0432\u0430\u0442\u0435\u043b\u0435\u0439
 AboutGeoServerPage.wiki          = \u0412\u0438\u043a\u0438
 
 AbstractCoverageStorePage.dataSrcName = \u041d\u0430\u0438\u043c\u0435\u043d\u043e\u0432\u0430\u043d\u0438\u0435 \u0438\u0441\u0442\u043e\u0447\u043d\u0438\u043a\u0430 \u0434\u0430\u043d\u043d\u044b\u0445
 AbstractCoverageStorePage.description = \u041e\u043f\u0438\u0441\u0430\u043d\u0438\u0435
 
-AbstractDataAccessPage.cantGetDataStoreFactory = Can't find the factory for the given data store. Make sure the \nappropriate plugin is installed
+AbstractDataAccessPage.cantGetDataStoreFactory = Can't find the factory for the given data store. Make sure the \
```
Comment (fgdrf) [inline src/web/core/src/main/resources/GeoServerApplication_ru.properties]: something went wrong here, "\n" replaced by "\" may result in a corrupt properties files

File: src/web/core/src/main/resources/GeoServerApplication_ru.properties [hunk 2]
```diff
@@ -142,7 +150,8 @@ CoverageAccessPage.imageio = ImageIO settings
 CoverageAccessPage.imageIOCacheThreshold = ImageIO Cache Memory Threshold (KB)
 CoverageAccessPage.poolSizeCheck = \u041c\u0430\u043a\u0441\u0438\u043c\u0430\u043b\u044c\u043d\u044b\u0439 \u0440\u0430\u0437\u043c\u0435\u0440 \u043f\u0443\u043b\u0430 \u043d\u0435 \u043c\u043e\u0436\u0435\u0442 \u0431\u044b\u0442\u044c \u043d\u0438\u0436\u0435, \u0447\u0435\u043c \u043e\u0441\u043d\u043e\u0432\u043d\u043e\u0439 \u0440\u0430\u0437\u043c\u0435\u0440 \u043f\u0443\u043b\u0430
 
-CoverageStoreEditPage.cantGetCoverageStoreFactory = Can't obtain the factory for the given Coverage Store. Make sure \nthe appropriate plugin is installed.
+CoverageStoreEditPage.cantGetCoverageStoreFactory = Can't obtain the factory for the given Coverage Store. Make sure \
```
Comment (fgdrf) [inline src/web/core/src/main/resources/GeoServerApplication_ru.properties]: see comment above

**Comment by kolosov-sergey** [comment]
That's how it was in GeoServerApplication.properties. I've removed \n to make files GeoServerApplication.properties and GeoServerApplication_ru.properties identical and to ease synchronization. You mean that there may be problems in transifex?

**Comment by fgdrf** [comment]
Some hints:
* default behavior for missing keys in i18 properties files is to take it from origin (e.g. GeoServerApplication.properties)
* its redundant if its listed in both and leave it "untranslated" in i18n properties file (e.g. AboutGeoServerPage.usefulLinks in GeoServerApplication_ru.properties) - therefore I would suggest to remove untranslated properties
* if comment these properties out (leading "#") you can import files to transifex and have no mess with the state (100% translated isn't real because of english phrases for copied properties)


I synched today from transifex (master) into my working copy with an ant-script (https://gist.github.com/fgdrf/6507435) and the configuration for transifex taken from https://gist.github.com/fgdrf/6507286

Looks good so far and I could add a commit to this pull request. Note that I can't verify the content itself (unicode reading is quite horrible ;))

HTH

**Comment by jodygarnett** [comment]
@fgdrf would you like me to merge this for the milestone release? Or is this work still underway ...

**Comment by fgdrf** [comment]
@jodygarnett Thanks for the offer. We are working on that and try to find out the current state of Russion translations. We are going to create an other pull later.

**Comment by kolosov-sergey** [comment]
@jodygarnett I've misunderstood workflow and now closing this pull request. I'll make another one later.

**Pull request closed without merging**